README.md

```diff
@@ -16,13 +16,48 @@ knife audit list
 knife audit show <id>
 knife notification list
 knife notification show <id>
+knife notification create <notification.json>
 knife rule list
 knife rule show <id>
+knife rule create <rule.json>
+```
+
+## Example data for creating rules and notifications:
+
+### Rules
+```
+{
+  "name": "New Rule Group 1",
+  "modified_by": "admin",
+  "with": {
+    "priority": 0
```

Comment:
I believe notifier_config needs it to sort: https://github.com/chef/notifier_config/blob/master/src/nc_obj_rule.erl

I wonder if a regex can pull it out so a user doesn't have to enter it in the rule and as a param?